src/PageExtension.php

```diff
@@ -2,59 +2,59 @@
 
```

Comment:
Might be good to rename it to something like `Superglue_PageExtension.php` so it is not too generic and displays the right intent, thoughts?